samples/compose/src/main/java/com/squareup/sample/compose/hellocomposebinding/HelloBindingActivity.kt

```diff
@@ -0,0 +1,53 @@
+package com.squareup.sample.compose.hellocomposebinding
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.material.MaterialTheme
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.squareup.workflow1.ui.ViewEnvironment
+import com.squareup.workflow1.ui.ViewRegistry
+import com.squareup.workflow1.ui.WorkflowLayout
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.compose.withCompositionRoot
+import com.squareup.workflow1.ui.renderWorkflowIn
+import kotlinx.coroutines.flow.StateFlow
+
+@OptIn(WorkflowUiExperimentalApi::class)
+private val viewRegistry = ViewRegistry(HelloBinding)
+
+@OptIn(WorkflowUiExperimentalApi::class)
+private val viewEnvironment =
+  ViewEnvironment(mapOf(ViewRegistry to viewRegistry)).withCompositionRoot { content ->
```

Comment:
You sure the default ViewRegistry entry is still required? Thought I fixed that.

Reply:
I am not sure, good catch.

Reply:
Actually this isn't the default, we're passing a specific binding through.